src/mmw/js/src/modeling/models.js

```diff
@@ -1003,8 +1005,18 @@ var ScenariosCollection = Backbone.Collection.extend({
             newModel = new ScenarioModel({
                 is_current_conditions: false,
                 name: this.makeNewScenarioName('Copy of ' + source.get('name')),
+                user_id: source.get('user_id'),
                 inputs: source.get('inputs').toJSON(),
-                modifications: source.get('modifications').toJSON()
+                inputmod_hash: source.get('inputmod_hash'),
+                modifications: source.get('modifications').toJSON(),
+                modification_hash: source.get('modification_hash'),
+                job_id: source.get('job_id'),
+                poll_error: source.get('poll_error'),
+                results: source.get('results').toJSON(),
+                has_fetched_results: source.get('has_fetched_results'),
```

Comment:
Should we be copying this @arottersman? The new scenario will have a new `taskModel` instance, could this interfere with that?